harper-core/tests/text/linters/Alice's Adventures in Wonderland.snap.yml

```diff
@@ -3340,6 +3340,16 @@ Message: |
 
 
 
+Lint:    Typo (31 priority)
+Message: |
+    2961 | watching the setting sun, and thinking of little Alice and all her wonderful
+    2962 | Adventures, till she too began dreaming after a fashion, and this was her
+         |                      ^~~ Use the infinitive marker `to` here instead of the adverb `too`, which indicates excess degree.
```

Comment:
You will be able to check the verb form to prevent this if my verb form PR makes it in. `to` can only be before the lemma/infinitive/citation form, `to begin` in this case.

Reply:
> You will be able to check the verb form to prevent this if my verb form PR makes it in.

Now possible!